app/bundles/ReportBundle/Crate/ReportDataResult.php

```diff
@@ -24,7 +27,56 @@ class ReportDataResult
      */
     private $types = [];
 
-    public function __construct(array $data)
+    /**
+     * @var array<mixed>
+     */
+    private $totals = [];
+
+    /**
+     * @var array<string>
+     */
+    private $columnKeys = [];
+
+    /**
+     * @var array<mixed>
+     */
+    private $graphs = [];
+
+    /**
+     * @var DateTime|null
+     */
+    private $dateFrom;
```

Comment:
Please use typed properties for all new properties
```suggestion
    private ?DateTime $dateFrom;
```